dbms/src/Storages/MergeTree/ReplicatedMergeTreeTableMetadata.cpp

```diff
@@ -78,6 +89,9 @@ void ReplicatedMergeTreeTableMetadata::write(WriteBuffer & out) const
     if (!ttl_table.empty())
         out << "ttl: " << ttl_table << "\n";
 
+    if (!ttl_move.empty())
```

Comment:
Why not at the end of the list?

Reply:
To group similar things together (`ttl` and `move_ttl`).